### DISCUSSION TITLE
Additional keywords for the ACG repository

### DISCUSSION THREAD

**Discussion by ftessier** [a-cosmology-group/acg#14] [A Cosmology Group]
Perhaps it would be useful to add additional relevant keywords to the `acg` repository (in the `About` section, top right), so that the repo becomes more visible to interested parties via keyword search.

**Comment by RedshiftDrift** [comment]
From Google performance tools, the top queries for cosmology.info are, from high to low:
- alternative cosmology,
- static universe,
- cosmology group,
- infinite static universe,
- dynamic nature of the universe,
- crisis in cosmology,
- alternative cosmology group,
- tension models,
- stationary universe,
- static universe theory,
- is the universe static,
- static universe model,
- redshift theory,
- static model of the universe,
- plasma cosmology.

These suggest keywords that could be added to the *About* section.

**Comment by RedshiftDrift** [comment]
Keywords added, and a social media image as well! [https://github.com/topics/cosmological-models](https://github.com/topics/cosmological-models)

**Comment by ftessier** [comment]
Looks good @marmetl! I did not even know you could add a social media image, cool. 😎

**Reply by RedshiftDrift** [reply]
Just discovered that by accident...  I've been clicking on everything 🤣
